libs/blocks/region-selector-loader/region-selector-loader.js

```diff
@@ -0,0 +1,108 @@
+import { createTag } from '../../utils/utils.js';
+
+const LOC_CONFIG = 'https://main--milo--adobecom.hlx.page/drafts/localization/configs/config.json';
+const HELIX_ADMIN = 'https://admin.hlx.page';
+
+const getLivecopies = async () => {
+  const livecopies = [];
+  const res = await fetch(LOC_CONFIG);
+  const json = await res.json();
```

Comment:
I've seen a typical pattern of: `if (!resp.ok) return;` would it be good to make that check here?